### PR TITLE
[seo-meta-tags][xs]: added seo meta tags for social media, dataset su…

### PR DESCRIPTION
**Pull request by Mikanebu**
* Added SEO meta tags for dataset summary and social media details

### CODE REVIEW

File: views/base.html [hunk 1]
```diff
@@ -10,6 +10,16 @@
   <meta name="description" content="{{title}}. Data online made easy. Share and find data, quickly and easily. Simple, solid, slick. Frictionless and fast. Built for data scientists and wranglers. Managed by Datopian and Open Knowledge International">
   <meta name="keywords" content="data,datasets,data publishing,data sharing,data hosting,data package,reference data,indicators">
   <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+  <meta name="og:title" content="DataHub">
```
Comment (anuveyatsu) [inline views/base.html]: I'm not sure how these meta tags work, but why we're not using dynamic titles? E.g., on https://datahub.io/core/finance-vix, I think the title should be "VIX - CBOE Volatility Index - DataHub"

File: views/base.html [hunk 2]
```diff
@@ -10,6 +10,16 @@
...
+  <meta name="og:url" content="https://datahub.io/">
```
Comment (anuveyatsu) [inline views/base.html]: Same here - we could use actual URLs instead of just host

File: views/base.html [hunk 3]
```diff
@@ -10,6 +10,16 @@
...
+  <meta name="og:description" content="https://datahub.io/">
```
Comment (anuveyatsu) [inline views/base.html]: Same here - I don't think URL is a description but readme snippet is.

File: views/base.html [hunk 4]
```diff
@@ -10,6 +10,16 @@
...
+  <meta name="og:image" content="/static/img/datahub-cube-edited.svg">
+  <meta name="twitter:card" content="player">
+  <meta name="twitter:creator	" content="@datahubio">
```
Comment (anuveyatsu) [inline views/base.html]: why there is a space after "creator"?

File: views/base.html [hunk 5]
```diff
@@ -10,6 +10,16 @@
...
+  <meta name="twitter:site" content="@datahubio">
+  {% if dataset.readmeSnippet %}
+    <meta name="summary" content="{{dataset.readmeSnippet}}">
```
Comment (anuveyatsu) [inline views/base.html]: what's the difference between summary and description?

File: views/base.html [hunk 6]
```diff
@@ -1,4 +1,4 @@
-<!DOCTYPE html>
+2<!DOCTYPE html>
```
[the remark this answers is not in the document]
Comment (Mikanebu) [reply views/base.html]: typo, removed

**Pull request merged**